src/pages/Doyun/Login/Login.scss

```
@@ -0,0 +1,49 @@
+.login {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 500px;
+  width: 400px;
+  margin: 25px auto;
+  border: 1px solid #bbbbbb;
+  background-color: #ffffff;
+  .mainLogo {
+    font-family: 'Lobster', cursive;
+    font-weight: 50;
+    font-size: 2.5rem;
+    margin-bottom: 40px;
+  }
+  input {
+    background-color: #fafafa;
+    border: 1px solid #ebebeb;
+    margin-bottom: 15px;
```

```suggestion
  border: 1px solid #bbbbbb;
  background-color: #ffffff;
  
  .mainLogo {
    font-family: 'Lobster', cursive;
    font-weight: 50;
    font-size: 2.5rem;
    margin-bottom: 40px;
  }
  
  input {
    background-color: #fafafa;
    border: 1px solid #ebebeb;
    margin-bottom: 15px;
```
가독성을 위해 새로운 셀렉터 가 시작될 때는 한줄씩 띄워주세요